EventCatalog/EventCatalogAPI.csproj

```diff
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
-    <DockerDefaultTargetOS>Windows</DockerDefaultTargetOS>
```

Comment:
DockerDefaultTarget is only Linux. This change will come from my end. This will be a conflict with my docker-compose.yml file.